### ISSUE TITLE
Add some example sound files to the README

### ISSUE THREAD

**Issue by NightMachinery** [veggiedefender/sounds#2]
Since it takes some effort to try the project out (Processing and its dependency management is a pain), having some demo files on the readme would be nice.

**Comment by veggiedefender** [comment]
Good idea! It's been a while so I'll have to set up Processing and its dependencies again :sweat_smile: but examples would definitely be valuable
